processing/src/main/java/io/druid/query/filter/DruidLongPredicate.java

```diff
@@ -28,5 +28,25 @@
 
   DruidLongPredicate ALWAYS_TRUE = input -> true;
 
```

Comment:
Is this class an `ExtensionPoint` or not? If yes, please add annotation and Javadocs to methods.

Reply:
not an extension point.